core/src/main/python/wlsdeploy/tool/util/filter_helper.py

```diff
@@ -30,20 +30,36 @@ def apply_filters(model, tool_type):
     _method_name = 'apply_filters'
 
     filter_applied = False
+    configuration = None
+
 
     try:
-        if os.path.isfile(__filter_file_location):
+        if model_context:
+            configuration = model_context.get_target_configuration()
+        filters_dictionary = {}
+        if configuration and 'model_filters' in configuration:
+            filters_dictionary = configuration['model_filters']
+            target_filter_path =  os.path.join(os.environ.get('WLSDEPLOY_HOME', ''), 'lib', 'target',
```

Comment:
Can the directory be lib/targets, similar to lib/injectors and lib/typedefs?

Reply:
done